### DISCUSSION TITLE
Is it possible to disable Rich without having to uninstall it?

### DISCUSSION THREAD

**Discussion by anthonyraf** [fastapi/typer#622] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
import typer

app = typer.Typer(add_completion=False)

@app.command()
def hello(name: str):
    typer.echo(f"Hello {name}"

if __name__ == '__main__':
    app()
```

### Description

When I use python myapp.py --help, the Rich panel appears, but I don't want it. I have already read the source code and I saw this:
https://github.com/fastapi/typer/blob/f83c66d539b95d00de74e7de6a14b5c53ae0bfe6/typer/main.py#L39-L47

I have considered installing only typer without rich by using poetry add typer instead of poetry add typer[all]. However, since my project itself uses rich, it automatically detects it as a dependency.
So, I believe it would be more efficient to add a enable_rich parameter, like this:
```python
import typer
app = typer.Typer(enable_rich=False)
...
```
Thanks

### Operating System

Linux

### Typer Version

0.9.0

### Python Version

3.11.2

### Additional Context

I use poetry as dependency manager

**Comment by JacobKochems** [comment, marked as answer]
Hello @anthonyraf, I made a pull request with an implementation of a feature similar to the one you described here.
Maybe you could write a review of my [PR](https://github.com/tiangolo/typer/pull/647) according to [this](https://typer.tiangolo.com/help-typer/#review-pull-requests) ?
